zstacklib/zstacklib/utils/upload_task.py

```
@@ -0,0 +1,350 @@
+                break
+
+        return ib
+
+    @staticmethod
+    def get_upload_param(headers):
+        """解析上传参数"""
+
+        def get_long_field(key, default=None):
+            v = headers.get(key, default)
+            try:
+                lv = long(v)
```

Ruff (0.13.1)

Undefined name `long`

(F821)